org-protocol-capture-html.el

```diff
@@ -79,7 +79,10 @@ Pandoc >= 1.16 deprecates `--no-wrap' in favor of
                 (sleep-for 0.2)
                 (cl-incf checked)))
             (if (and (zerop (process-exit-status process))
-                     (not (string-match "--no-wrap is deprecated" (buffer-string))))
+                     (not (string-match
+                           (rx (: (1+ blank) "--no-wrap"
```

Comment:
If I may nitpick even harder (and perhaps fix a bug, since I don't think that will match the space after `--no-wrap`), I think this is all that's needed:
```el
(rx "--no-wrap " (or "is deprecated" "has been removed"))
```